docs/AboutUs.md

```diff
@@ -9,51 +9,13 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Hoang Le Tri Cuong
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/ernestcuong.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
-
-* Role: Project Advisor
-
-### Jane Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
-
-* Role: Team Lead
-* Responsibilities: UI
-
-### Johnny Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](https://https://github.com/ErnestCuong)]
```

Comment:
Should the GitHub address here be `https://github.com/ErnestCuong` instead?

Reply:
You are correct. My bad. When I copied the address there was no `https://` but when pasting it somehow it appeared.